src/admin/components/SearchPage.js

```diff
@@ -30,9 +30,9 @@ export default class AdminSearchPage extends React.Component {
     } = this.props;
     const pathname = '/search/';
     const paginator = hasSearchParams && count > 0 ?
-      <Paginate count={count} currentPage={page}
+      (<Paginate count={count} currentPage={page}
```

Comment:
I feel like in the past @mstriemer has said this should look more like:

```js
const paginator = hasSearchParams && count > 0 ? (
  <Paginate [...] />
) : [];
```

Or at least it'd make more sense to me. I know it was like that when you got here, and it's better than before, so I leave it up to you 😄